tests/testthat/test-s3-methods-operator.R

```diff
@@ -0,0 +1,38 @@
+make_cases = function() {
+  tibble::tribble(
+    ~.test_name, ~fn,
+    "add", `+`,
+    "sub", `-`,
+    "div", `/`,
+    "floor_div", `%/%`,
+    "mul", `*`,
+    "mod", `%%`,
+    "pow", `^`,
+  )
+}
```

Comment:
didn't we already have some parametric tests for that in test-expr_expr.R? It would be good to keep them in a single place